stratum/hal/bin/barefoot/BUILD

```diff
@@ -166,6 +166,8 @@ pkg_tar(
     ],
 )
 
+sde_bsp_debian_deps = ["libusb-1.0-0"]
```

Comment:
This should really go into `bfsde.build`, but sharing variables requires using `.bzl` files, which might be overkill for a single instance.

Alternatively, we could package the SDE into a deb with the correct deps, and depend on that later. @bocon13 floated that idea before.